data_dir/scylla-dash-per-server-nemesis.master.json

```diff
@@ -25,6 +25,20 @@
                   ],
                   "type": "tags"
                 },
+                {
+                  "datasource": "-- Grafana --",
+                  "enable": false,
+                  "hide": false,
+                  "iconColor": "#aaacad",
+                  "limit": 100,
+                  "matchAny": true,
+                  "name": "Events[STARRED]",
```

Comment:
If we want to create a specific filter, we can filter based on other tags, force the severity into that.
if this filter is only for showing reactor stalls ?

Reply:
@roydahan's idea was to have a new severity for "reactor stalls", but @enaydanov didn't like the idea, so we came up with this idea of being able to "star" ("pin", "mark"...) specific events, to be able to work with them, as in the email reports

Reply:
same as above